Sources/App/Views/PackageController/PackageShow+View.swift

```diff
@@ -329,3 +354,21 @@ extension PackageShow {
         }
     }
 }
+
+extension FundingLink {
+    var label: String {
+        switch platform {
+            case .communityBridge: return "LFX Mentorship"
+            case .customUrl: return URL(string: url)?.host?.removingPrefix("www.").removingPrefix("blog.") ?? url
```

Comment:
There are a couple of cases in the existing data where this doesn't get the subdomain and while this CTA doesn't look as good with them left in, I also don't want to start getting into adding special cases for "jaywcjlove" and "spezi.sites".